lib/plugins/block_actions.js

```diff
@@ -35,6 +35,7 @@ function inject (bot, { version }) {
     if (bot.supportFeature('doesntHaveChestType')) {
       const facing = parseChestMetadata(chestBlock).facing
 
+      if(facing == null) return;
```

Comment:
Probably better to do `return 'single'` , otherwise `'chestLidMove'` won't be emitted in these cases

Reply:
Using strict equals causes the original crash to still happen, added return `single` and removed the ; however it didn't pass.

Reply:
Then it's because `facing` is `undefined`

You can use `if(!facing) return 'single'` to catch it when it's undefined/null